packages/tutorials/election-dapp/front-end/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@tutorials/election",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@kadena/chainweb-node-client": "^0.1.1",
+    "@kadena/client": "^0.1.5",
+    "@kadena/pactjs-cli": "^0.0.7",
+    "@kadena/types": "^0.0.3",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^13.2.1",
+    "@types/jest": "^27.0.1",
+    "@types/node": "^16.7.13",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-responsive-modal": "^6.4.1",
+    "react-scripts": "5.0.1",
+    "spinners-react": "^1.0.7",
+    "typescript": "^4.4.2"
+  },
```

Comment:
Can we split up into `dependencies` and `devDependencies`, and replace them with `@rush`-libraries where applicable

Reply:
Yes I'll split them up. This is a basic create-react-app to demonstrate `@kadena/client` usage as part of the tutorial, why do we want to use rush here?

Reply:
Ah, I see. So it's not actually being compiled here. Then you can ignore my other `@rush` comments as well.